pkg/backup/http.go

```diff
@@ -87,5 +87,9 @@ func (b *Backup) serveSnap(w http.ResponseWriter, r *http.Request) {
 	if r.Method == http.MethodHead {
 		return
 	}
-	http.ServeFile(w, r, path.Join(b.backupDir, fname))
+
+	_, err = io.Copy(w, rc)
+	if err != nil {
+		logrus.Errorf("failed to write backup to %s: %v", r.RemoteAddr, err)
```

Comment:
actually we cannot write http header as long as we ever write one byte in the body.